assets/src/edit-story/components/helpCenter/menu/header.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import { themeHelpers } from '../../../../design-system';
+
+const Panel = styled.div`
+  padding: 24px 0 16px 0;
+  border-bottom: 1px solid ${({ theme }) => theme.colors.bg.tertiary};
+`;
+
+const Heading = styled.h4`
+  ${themeHelpers.expandTextPreset(({ label }, { MEDIUM }) => label[MEDIUM])}
+  line-height: 32px;
+  margin: 0 0 8px 0;
+`;
```

Comment:
heck! this one's asking for a size 16px h4 when our headline xxs and xs are 14 and 20 :( otherwise you could just do `<Headline as="h4" size={XX_S}`>` and adjusted the bottom margin.